src/webapi/child_node.rs

```diff
@@ -0,0 +1,21 @@
+use webcore::reference_type::ReferenceType;
+
+/// The `ChildNode` interface contains methods that are particular to `Node`
+/// objects that can have a parent.
+///
+/// You most likely don't want to `use` this directly; instead
+/// you should `use stdweb::traits::*;`.
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode)
+// https://dom.spec.whatwg.org/#interface-childnode
+pub trait IChildNode: ReferenceType {
+    /// The `ChildNode.remove()` method removes the object from the tree it belongs to.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove)
+    // https://dom.spec.whatwg.org/#dom-childnode-remove
```

Comment:
This should link to the IDL, that is: https://dom.spec.whatwg.org/#ref-for-dom-childnode-remove